packages/apps-v5/src/commands/ps/index.js

```diff
@@ -68,13 +68,25 @@ async function printAccountQuota(context, heroku, app, account) {
   let appHours = Math.floor(appQuotaUsed / 60)
   let appMinutes = Math.floor(appQuotaUsed % 60)
 
-  cli.log(`Free dyno hours quota remaining this month: ${hours}h ${minutes}m (${percentage}%)`)
-  cli.log(`Free dyno usage for this app: ${appHours}h ${appMinutes}m (${appPercentage}%)`)
-  cli.log('For more information on dyno sleeping and how to upgrade, see:')
-  cli.log('https://devcenter.heroku.com/articles/dyno-sleeping')
-  cli.log()
+  if (app.process_tier === 'eco') {
+    cli.log(`Eco dyno hours quota remaining this month: ${hours}h ${minutes}m (${percentage}%)`)
+    cli.log(`Eco dyno usage for this app: ${appHours}h ${appMinutes}m (${appPercentage}%)`)
+    cli.log('For more information on Eco dyno hours, see:')
+    cli.log('https://devcenter.heroku.com/articles/eco-dyno-hours')
+    cli.log()
+  }
+
+  if (app.process_tier === 'free') {
```

Comment:
minor nit, should this be an `else if`, since if it can't be both `eco` AND `free`?

Reply:
I think it should be fine. We're just making 2 cases depending on the only 2 options this function should be able to output from `app.process_tier` .